python/mlx/utils.py

```diff
@@ -143,3 +145,103 @@ def tree_unflatten(tree):
         return l
     else:
         return {k: tree_unflatten(v) for k, v in children.items()}
+
+
+def atleast_1d(*values):
```

Comment:
These should be implemented in C++ not Python and then bound to Python via Pybind (the same as all the rest of our ops).